### PR TITLE
add papparich

### PR DESCRIPTION
**Pull request by MASangys**

### CODE REVIEW

**Comment by ArenaL5** [comment]
Sorry, I wasn't following the talk at #3904. I don't want to leave repeated comments on each pull request.

Most of your Wikidata entries have issues. We'll try to guide you through #3904, and after that, you'll be able to correct your other pull requests.

**Comment by MASangys** [comment]
Thanks for reply, I had expand a little bit of the wikipedia entry. No worries cuz Malay Wikipedia is not tight like English Wikipedia. And you know it take times to write a A+ article, I will try to expand more in the future. Buf now my mission is to fillup the empty chain store data in Malaysia. Wikidata for PappaRich also had been improved, so there is no problem for this PR right. Alt name I forget it and if you dont mind please fix it and merge this PR thank you.

**Comment by bhousel** [comment]
@MASangys .. I appreciate your enthusiasm for adding brands, but it sounds like you should slow down and try to do a more thorough job with your contributions.  

_There is no rush to get all these brands added._  I'd appreciate it if you would make sure the files you are adding have the correct syntax, make sure your wikidata and wikipedia contributions are complete, and please don't ask other contributors to finish your work for you.

**Comment by MASangys** [comment]
@bhousel I am not famillier with GitHub and thanks for your feedback. I will try to improve wikidata for those brands. For syntex I had did some checkup at JSON and only paparich I forget a comma,  since I copy the script from pther brand and edit it so it might have some error. Wikipedia is ok and as I said I will expand more.

''and please don't ask other contributors to finish your work for you.''

Sorry for that, but I am learning on GitHub too, so my work might be imprefect, but I will try to enhance that based on feedback. Definitely everyone dont want to waste thier time on something that are not related to them. So I do really appreciate if someone help to do some of the work,  but just like what I said,  most of the work I will finish it,  so again thanks for your feedback,  I will try to be responsible too.

**Comment by ArenaL5** [comment]
Okay, your [Wikidata entry](https://www.wikidata.org/wiki/Q95721282) is much better. There are still some things to improve:
- PappaRich's website says there are several restaurants, so it's probably an instance of [**restaurant chain** (Q18534542)](https://www.wikidata.org/wiki/Q18534542), not [**restaurant** (Q11707)](https://www.wikidata.org/wiki/Q11707). Wikidata is very precise.
“Restaurant chain” hasn't been translated to Malay, so you need to write it in another language, or just copy-paste `Q18534542`. The validator will work this way.

- You can, of course, [translate it yourself](https://www.wikidata.org/wiki/Q18534542) if you want.

- Please change the colour of the logo. I can't see white letters on a white background.
The logos at the bottom of the website are yellow and brown, you can crop one of those.

- We'd appreciate it if you could add [**Facebook ID** (P2013)](https://www.wikidata.org/wiki/Property:P2013). We'd also ask for their [**Twitter username** (P2002)](https://www.wikidata.org/wiki/Property:P2002), but they don't use Twitter, don't they?
The reason for this is because we have a script that checks those pages, so we can find some mistakes.
Some programs use them to fetch the brand's logo.

I can't read your Wikipedia and I'm not familiar with their practices, but if it's anything like the Spanish and English branches, there is [a translation of Wikipedia:Stub](https://ms.wikipedia.org/wiki/Wikipedia:Rencana_tunas) you might be interested in reading. (Also _maybe_ add an [infobox](https://ms.wikipedia.org/wiki/Templat:Infobox_company)?)

**Comment by MASangys** [comment]
fixed

**Comment by ArenaL5** [comment]
Okay, your Wikidata page is almost perfect. I'm sorry I didn't notice this earlier:

PappaRich

The only important thing missing is the name of the restaurant. By mistake, you only filled the German name with an English description.

You should at the very least:
- fill in the name and description in Malay (so that Malay speakers can find it easily)
- fill in the name and description in English (because most people in this kind of projects speak English, and also because there's an Australian branch),
- and remove the English description from the German name

[I noticed that they opened a restaurant at Taiwan.](https://papparich.com.my/milestones/) If you can find the official name of the store in Chinese, please add it, too. If you can't speak Chinese you can leave the Chinese description blank.

**Comment by MASangys** [comment]
Malay should be

 Restoran rantai di Malaysia

other had added. I dont know why I cannot input ms in the field.

**Comment by UKChris-osm** [comment]
Hi @MASangys,

After **"alt_name": "PappaRich Malaysian Restaurant"** you are missing a single comma, which means the next line in this entry is creating an error.

Adding a comma, at the end of **"alt_name": "PappaRich Malaysian Restaurant",** will fix this, and can be easily done with the fork that you already have. Once you have added the comma and saved it, this PR should update itself to pass the Travis CL build test.

**Comment by ArenaL5** [comment]
> Malay should be
> 
> Restoran rantai di Malaysia
> 
> other had added. I dont know why I cannot input ms in the field.

There are ways to input names on any language, but all of them are annoying.

Probably the easiest way is to log in into your account, then click on the language selector near the top center of the screen:

Wikidata_languages

Then, you can type `ms` in the search bar to select Malay:

ms_malay

Click Bahasa Melayu and wait for the page to reload. Wikidata will be translated to Malay and the item at the top of the label list will be in Malay.

With a bit of luck, Wikidata will remember you speak Malay forever.

File: brands/amenity/restaurant.json [hunk 1]
```diff
@@ -1827,6 +1827,17 @@
       "cuisine": "pizza",
       "name": "Papa Gino's"
     }
+  },
+    "amenity/restaurant|PappaRich": {
+    "tags": {
+      "alt_name": "PappaRich Malaysian Restaurant",
```
Comment (ArenaL5) [inline brands/amenity/restaurant.json:1833]: While there are no syntax errors, you might be better off removing this line altogether.

```suggestion
```

Remember that anything within `"tags": { }` will be added to new points at OpenStreetMap by default. Every time a mapper creates a node for PappaRich, or uses the validator to add the wikidata, they will also add that `alt_name`.

“Malaysian Restaurant” doesn't seem to be part of the name in any country, and `"cuisine": "malaysian"` and `"amenity": "restaurant"` already tag this as a Malaysian-style restaurant. Are you sure this name is worth keeping?

**Malaysian mappers have the last word on this, not us, not you.**

Comment (MASangys) [reply brands/amenity/restaurant.json:1833]: Ok, I saw it at this tag
https://www.openstreetmap.org/node/6296999656
So mean that if I remote this alt name, then this xhain restaurant in Syndey also need to do so?

Comment (UKChris-osm) [reply brands/amenity/restaurant.json:1833]: It seems that there is only that single node in Australia that has the "alt_name" attached to it @MASangys, so I would actually remove it.

Comment (MASangys) [reply brands/amenity/restaurant.json:1833]: Ok, just removed it. No problem

**Comment by UKChris-osm** [comment]
@MASangys, would you agree that the Facebook used on the Wikidata would be better served as "_papparichgroup_" or are you happy with the one currently listed?

I only ask as I think the one currently listed is the Facebook page of a single restaurant?

**Comment by MASangys** [comment]
No problem, please merge this PR

**Comment by ArenaL5** [comment]
OK. :+1: 
Push a commit removing the `alt_name`, and I'll merge it.

**Comment by UKChris-osm** [comment]
All done @MASangys, thank you :)

**Pull request merged**